### PR TITLE
feat(layer-selector): add optional opacity slider and some fixes and refactors

### PR DESCRIPTION
**Pull request by stdavis**
Add an optional opacity slider stemming from [this request](https://github.com/agrc/wfrc-rtp-projects/issues/64).

Some fixes and refactors including a simplification of the layer item layout and styling.

Before:
image

After:
image

### CODE REVIEW

File: packages/layer-selector/src/LayerSelector.jsx [hunk 1]
```diff
@@ -316,14 +340,12 @@ const LayerSelector = (props) => {
       ) || [];
     let overlays = props.overlays || [];
 
-    let hasDefaultSelection = false;
```
Comment (steveoh) [inline packages/layer-selector/src/LayerSelector.jsx:319]: **question**: do default base map and overlay selections still work?

File: packages/layer-selector/src/LayerSelector.jsx [hunk 2]
```diff
@@ -449,6 +478,21 @@ const LayerSelector = (props) => {
...
+                min="0"
+                max="100"
+                step="1"
+                value={opacity * 100}
+                onChange={(event) =>
+                  setOpacity(parseFloat(event.target.value) / 100.0)
+                }
+              />
+            </>
+          ) : null}
```
Comment (steveoh) [inline packages/layer-selector/src/LayerSelector.jsx:495]: **praise**: nice!

**Review by steveoh — COMMENTED** [review]
Looks pretty good to me so far.

File: packages/layer-selector/src/LayerSelector.jsx (hunk 1 above)
Comment (stdavis) [reply packages/layer-selector/src/LayerSelector.jsx:319]: Yes. I added some stories for testing.

**Review by steveoh — DISMISSED** [review]

**Comment by steveoh** [comment]
This might need a rebase and force push

**Pull request merged**